lib/yt/collections/videos.rb

```diff
@@ -33,13 +33,16 @@ def attributes_for_new_item(data)
 
       def eager_load_items_from(items)
         if included_relationships.any?
-          include_category = included_relationships.delete(:category)
-          include_claim = included_relationships.delete(:claim)
-          included_relationships.append(:snippet).uniq! if include_category || include_claim
+          associations = [:claim, :category]
+          if (included_relationships & associations).any?
+            included_relationships.append(:snippet).uniq!
+          end
 
           ids = items.map{|item| item['id']['videoId']}
-          parts = included_relationships.map{|r| r.to_s.camelize(:lower)}
-          conditions = {id: ids.join(','), part: parts.join(',')}
+          parts = (included_relationships - associations).map do |r|
+            r.to_s.camelize(:lower)
+          end
+          conditions = { id: ids.join(','), part: parts.join(',') }
```

Comment:
I used to have rubocop hooked up in some other projects and by default it demanded me to put in those spaces between curly braces lol.